### PR TITLE
Add ByteSlice Method for embed etcd

### PR DESCRIPTION
**Pull request by letian-jiang**
Add []byte methods for embed etcd
Related to Issue #10468

Signed-off-by: Letian Jiang <letian.jiang@zilliz.com>

### CODE REVIEW

File: internal/kv/etcd/embed_etcd_kv.go [hunk 1]
```diff
@@ -324,6 +499,25 @@ func (kv *EmbedEtcdKV) MultiSaveAndRemoveWithPrefix(saves map[string]string, rem
 	return err
 }
 
+// MultiSaveBytesAndRemoveWithPrefix saves kv in @saves and removes the keys with given prefix in @removals.
+func (kv *EmbedEtcdKV) MultiSaveBytesAndRemoveWithPrefix(saves map[string][]byte, removals []string) error {
+	ops := make([]clientv3.Op, 0, len(saves))
```
Comment (congqixia) [inline internal/kv/etcd/embed_etcd_kv.go:504]: Maybe `len(saves) + len(removals)`

Comment (letian-jiang) [reply internal/kv/etcd/embed_etcd_kv.go]: Yes

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**